lib/rubocop/cop/style/if_unless_modifier_of_if_unless.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for if and unless statements used as modifers of other if or
+      # unless statements
```

Comment:
Add here some examples. You can refer to other cops to see the typical class docs we employ.